badgerdb/badgerdb.go

```diff
@@ -97,6 +98,38 @@ type Options struct {
 	// Directory for storing the DB files.
 	// Optional ("BadgerDB" by default).
 	Dir string
+	// Uses in-memory storage, ignores file-specified options.
+	// Optional (false by default).
+	InMemory bool
+	// When SyncWrites is true all writes are synced to disk.
+	// Optional (false by default).
+	SyncWrites bool
```

Comment:
Is this option ignored when `InMemory` is `true`? If yes, should that be mentioned here in the Godoc, similar to what's mentioned in the Godoc of `Truncate` (ignored when ReadOnly is true)?